requirements.txt

```diff
@@ -11,4 +11,7 @@ setuptools>=42
 wheel
 pytest
 orjson
-patchelf; sys_platform == 'linux'
\ No newline at end of file
+patchelf; sys_platform == 'linux'
+
+-f https://github.com/makslevental/pyframe_eval/releases/expanded_assets/0.0.1
+pyframe_eval
```

Comment:
Can we copy a version of it into PI ? Also you probably want to set the license for that.  Ideally Apache 2.0 is best for PI.

Reply:
you/we/I can copy but do you really want it here? It's hairy cpython internals - might as well pawn maintenance off on me into perpetuity.